target/linux/ar71xx/image/ubnt.mk

```diff
@@ -133,6 +133,14 @@ define Device/ubnt-unifiac-lite
   BOARDNAME := UBNT-UF-AC-LITE
 endef
 
+define Device/ubnt-unifiac-mesh
+  $(Device/ubnt-unifiac)
```

Comment:
You could just put here `$(Device/ubnt-unifiac-lite)` instead and then overwrite just one variable which is different, the `DEVICE_TITLE`. Result: 3 lines less.